README.md

```diff
@@ -369,12 +375,16 @@ customer_ingress:
     alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
     alb.ingress.kubernetes.io/healthcheck-path: /health
     alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}, {"HTTPS": 3030}]'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
     alb.ingress.kubernetes.io/scheme: internal
     alb.ingress.kubernetes.io/subnets: <VPC_PRIVATE_SUBNET>
     alb.ingress.kubernetes.io/tags: env=prod
     alb.ingress.kubernetes.io/target-type: ip
-  host: sombra-customer.my-domain.com
+  hosts:
+    - host: sombra-transcend.my-domain.com
```

Comment:
Can this be `sombra-transcend-ingress` and `sombra-customer-ingress`? We use those terms everywhere in the docs and internally, so it would be nice to be consistent

Reply:
ditto

Reply:
Customers use these names as references -- I understand this is an example, but we should still keep it consistent IMO